### DISCUSSION TITLE
Kartenbild herunterladen

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#140] [General]
Werkzeug Kartenbild herunterladen
Wenn man beim Rechteck aufziehen zu weit an den Kartenrand kommt bleibt die Box hängen.
Der Download wird zwar gemacht aber man muss ein refresh der Seite machen um das Rechteck wieder los zu werden.

![image](https://github.com/gis-eni/webgis-forum/assets/124671123/062e02ad-3db3-4a6b-b29b-7f1fd67ac8ab)

**Comment by gis-eni** [comment]
Da hast du recht, das Verhalten ist komisch. Der Bug ist ab Build 5.24.501 behoben
